src/generate-elements.ts

```diff
@@ -85,7 +86,7 @@ function serializeElement(
   const events = resolvedElement.events.map(
       e => ({
         name: e.name,
-        description: e.description,
+        description: e.description || '',
```

Comment:
Got it, thanks for the explanation. I still feel like `undefined` is more true here and will be an easier interface for users to interact with, but I do see the value in simplifying our types and elements.json format. I'll leave the decision up to you :)